imports/plugins/core/checkout/client/containers/completedOrderContainer.js

```diff
@@ -33,6 +33,27 @@ handlers.handleDisplayMedia = (item) => {
   return false;
 };
 
+handlers.onCancelOrderClick = (order) => {
+  Alerts.alert({
+    title: "Cancel Order",
+    type: "question",
+    text: "Are you sure you want to cancel this order?",
+    showCancelButton: true
+  }, (isConfirm) => {
+    if (isConfirm) {
+      Meteor.call("wallet/cancelOrder", order, (err, res) => {
+        if (res === 2) {
```

Comment:
Please assign number `2` to a constant or create an enum for responses e.g response = { ERROR: 2 }. It's easier to read and manage.